commands/auth.ts

```diff
@@ -198,12 +221,16 @@ exports.handler = async options => {
   ]);
 
   const accountId = getAccountId(accountName);
-  await trackAuthAction('auth', authType, TRACKING_STATUS.COMPLETE, accountId);
+  await trackAuthAction('auth', authType, TRACKING_STATUS.COMPLETE, accountId!);
```

Comment:
nit: now that you've updated `accountId` to accept `undefined`, you don't need to assert that it exists